src/textual/features.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+import sys
+
+from typing import cast
+
+if sys.version_info >= (3, 8):
+    from typing import Final, Literal
+else:
+    from typing_extensions import Final, Literal
+
+FEATURES: Final = {"devtools", "debug"}
```

Comment:
We could also do this, which is less elegant but avoids repeating our feature flags: :slightly_smiling_face: 
(because it's not very elegant to rely on this `__arg__` property of Literals, choosing to use it or not is quite subjective ^^)

```py
FeatureFlag = Literal["devtools", "debug"]

FEATURES: Final = set(FeatureFlag.__args__)

Reply:
Yeah, I don't like the repetition either. But the dunder attribute in a typing var makes me nervous.